spec/javascripts/PaymentGateways/StripeFormWrapper.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { StripeElementsForm } from 'PaymentGateways/components/StripeFormWrapper'
+import { mount } from 'enzyme'
+
+const props = {
+  stripePublishableKey: 'abcd',
+  setupIntentSecret: 'efgh',
+  billingAddressDetails: {
+    line1: '1002 Avenue de los Mexicanos',
+    line2: '',
+    city: 'South Park',
+    state: 'CO',
+    postal_code: '80440',
+    country: 'US'
+  },
+  successUrl: '/Broflovski/Residence',
+  isCreditCardStored: false
+}
+
+it('should render properly', () => {
+  const wrapper = mount(<StripeElementsForm { ...props } />)
+  expect(wrapper).toMatchSnapshot()
```

Comment:
This is redundant and I think too expensive. We should just check that the component gets rendered with a simple and quick assertion.
```suggestion
```